python/ray/data/impl/stats.py

```diff
@@ -77,7 +78,7 @@ def build(self, final_blocks: BlockList) -> "DatasetStats":
         return stats
 
 
-@ray.remote(num_cpus=0, placement_group=None)
+@ray.remote(num_cpus=0)
```

Comment:
Per @rkooo567, PG doesn't work with `num_cpus=0` tasks or actors.

Reply:
Hmm what are the `num_cpus=0` + placement groups semantics, exactly? Does a `num_cpus=0` actor never get placed in a placement group, or can it always be placed in a placement group, regardless of whether it requests the placement group?

Reply:
Ping here @jjyao @rkooo567, what are the current placement group semantics for `num_cpus=0` tasks/actors?

Reply:
I tried last time, and it doesn't use pg in this case.

Reply:
I don't think this is a blocker, since (1) this is aiming to solve a deadlock problem that zero-CPU actors aren't causing, and (2) the long-term goal is to have Datasets tasks/actors be able to schedule within AND outside of the placement group (tolerate the placement group taint but don't require to be placed in the placement group).

As long as zero-CPU actors will still be schedulable on nodes whose CPU cores are all taken up by placement groups (right?), I think that this is fine. cc @ericl

Reply:
Yeah, it doesn't matter as long as this actor gets placed somewhere.